metrics/meteor/README.md

```diff
@@ -0,0 +1,111 @@
+# Metric Card for METEOR
+
+## Metric description
+
+METEOR (Metric for Evaluation of Translation with Explicit ORdering) is a machine translation evaluation metric, which is calculated based on the harmonic mean of precision and recall, with recall weighted more than precision. 
+
+METEOR is based on a generalized concept of unigram matching between the machine-produced translation and human-produced reference translations. Unigrams can be matched based on their surface forms, stemmed forms, and meanings. Once all generalized unigram matches between the two strings have been found, METEOR computes a score for this matching using a combination of unigram-precision, unigram-recall, and a measure of fragmentation that is designed to directly capture how well-ordered the matched words in the machine translation are in relation to the reference. 
+
+
+## How to use 
+
+METEOR has two mandatory arguments:
+
+`predictions`: a list of predictions to score. Each prediction should be a string with tokens separated by spaces.
+
+`references`: a list of references for each prediction. Each reference should be a string with tokens separated by spaces.
+
+It also has several optional parameters:
+
+`alpha`: Parameter for controlling relative weights of precision and recall. The default value is `0.9`.
+
+`beta`: Parameter for controlling shape of penalty as a function of fragmentation. The default value is `3`.
+
+`gamma`: The relative weight assigned to fragmentation penalty. The default is `0.5`. 
+
+Refer to the [METEOR paper](https://aclanthology.org/W05-0909.pdf) for more information about parameter values and ranges.
+
+```python
+>>> from datasets import load_metric
+>>> meteor = load_metric('meteor')
+>>> predictions = ["It is a guide to action which ensures that the military always obeys the commands of the party"]
+>>> references = ["It is a guide to action that ensures that the military will forever heed Party commands"]
+>>> results = meteor.compute(predictions=predictions, references=references)
+```
+
+You will need [NLTK](https://www.nltk.org/) version >=3.7 for METEOR score to work.
```

Comment:
Are you sure about this requirement? Looking at the script code, we did some fixes to make it work for different NLTK versions, like 3.6.4, 3.6.5, 3.6.6...

Reply:
it wasn't working for me when I had version 3.6.4...